src/photos/styles/toolbar.styl

```diff
@@ -5,6 +5,8 @@
     margin-left auto
 
     a
+        &.pho-action-newalbum
+            background embedurl("../assets/icons/icon-album-add.svg") em(16px) center no-repeat
```

Comment:
That's a different thing than the menu rewrite isn't it?

Reply:
Right, but this button needed to be corrected too because of the last cozy-ui refactoring

Reply:
👍